### ISSUE TITLE
GrpcServer can't be mounted in a Vert.x Web router 

### ISSUE THREAD

**Issue by tsegismont** [eclipse-vertx/vertx-grpc#84]
The [documentation](https://github.com/eclipse-vertx/vertx-grpc/blame/main/vertx-grpc-server/src/main/asciidoc/server.adoc#L42) and the [4.3 release blog post](https://vertx.io/blog/whats-new-in-vert-x-4-3/#vertx-grpc) indicate that a `GrpcServer` can be mounted in a Vert.x Web router.

In fact, it is not possible because the gRPC server is a `Handler<HttpServerRequest>` not a `Handler<RoutingContext>`. So this blog post example is wrong:

```java
GreeterGrpc.GreeterImplBase service = new GreeterGrpc.GreeterImplBase() {
  @Override
  public void sayHello(HelloRequest request, StreamObserver<HelloReply> responseObserver) {
    responseObserver.onNext(HelloReply.newBuilder().setMessage("Hello " + request.getName()).build());
    responseObserver.onCompleted();
  }
};

GrpcServer grpcServer = GrpcServer.server(vertx);
GrpcServiceBridge serverStub = GrpcServiceBridge.bridge(service);
serverStub.bind(grpcServer);

router.consumes("application/grpc").handler(grpcServer); // does not compile
```

**Comment by tsegismont** [comment]
@vietj did I miss anything?

**Comment by vietj** [comment]
I would like to avoid this dependency of vertx-grpc over vertx-web for various reasons.

Instead we should provide the code `return rc -> server.handle(rc.request())` in the documentation, alternatively we could consider having a dependency of vertx-web onto vertx-grpc, but I think that is not necessary.

**Comment by tsegismont** [comment]
OK, let's fix the documentation and the blog post

**Comment by tsegismont** [comment]
Fixed by 31d21d8b9709e37ab67c78d03ee48db8c9ed12bf
